frontend/web/components/users-permissions/OrganisationUsersTable/components/LastLogin.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react'
+import moment from 'moment'
+
+interface LastLoginProps {
+  lastLogin: string
+}
+
+const LastLogin: React.FC<LastLoginProps> = ({ lastLogin }) => {
+  const formatLastLogin = (
+    last_login: string | undefined,
+  ): {
+    relativeDate: string
+    date?: string
+  } => {
+    if (!last_login) return { relativeDate: 'Never' }
+
+    const loginMoment = moment(last_login)
+
+    if (!loginMoment.isValid()) {
+      return { relativeDate: 'Never' }
+    }
+
+    const diff = moment().diff(loginMoment, 'days')
+
+    if (diff === 0) return { relativeDate: 'Today' }
+    if (diff === 1) return { relativeDate: 'Yesterday' }
+
+    return {
+      date: diff > 30 ? moment(last_login).format('Do MMM YYYY') : undefined,
```

Comment:
As before, we show the full date only if > 30 days
<img width="692" height="94" alt="image" src="https://github.com/user-attachments/assets/1b143dd9-7ec6-4ce7-80c8-b0fc7a6eb32d" />
<img width="620" height="59" alt="image" src="https://github.com/user-attachments/assets/7b850754-bbbb-418f-8fb3-dcc2a8232c7e" />